### DISCUSSION TITLE
MQTT documentation

### DISCUSSION THREAD

**Discussion by jeroenterheerdt** [awawa-dev/HyperHDR#492] [General]
I was intrigued by the new MQTT settings but couldn't find info on how to use it, where does it listen to, what commands to send? Is anything available about that?

**Comment by awawa-dev** [comment]
Hi
Currently  it generally exposes JSON api (the designer is in the Advanced->JSON APi tab) to MQTT so you can sent various commands to HyperHDR. The port configuration is on the HyperHDR MQTT section in the network tab.
